Dockerfile

```diff
@@ -157,6 +157,11 @@ RUN ARCH=$(arch | sed s/aarch64/arm64/) \
     rm requirements.txt; \
     fi
 
+# Extract ScanCode license texts to a directory
+RUN scancode-license-data --path=/opt/scancode-license-data \
+    && find /opt/scancode-license-data -type f ! -name "*.LICENSE" -exec rm -f {} + \
```

Comment:
Nit: Please use the more speaking `-not` instead of `!` (as the latter also has a special meaning in Bash).

Reply:
I had a second thought on it: If _scancode-license-data_ is the "API" way to access the license texts, then I don't want to put implementation-specific knowledge into the Dockerfile like what files I expect to be there, and which ones to delete. Because that might change over time, even without notice.

Reply:
> If scancode-license-data is the "API" way to access the license texts

Well, the `scancode-license-data` command was originally designed to dump the data for a static website. In that sense, we're just "misusing" it here, and it's by-design that it does more than what we need.

Reply:
> Because that might change over time, even without notice.

True. But to guard against that, I was planning to make a follow-up change that runs `ort requirements` as part of the Docker image build, to fail if ORT cannot find any licenses. We already were doing that at some point, but apparently it got dropped during some Docker image build process refactoring.